### DISCUSSION TITLE
[A-L] (2023/24) Foglio 5 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#304] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-11-04 181223](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/d2ef6a55-d378-41c8-a1ac-84d9d0ae3dc0)

**Comment by IonutCicio** [comment]
/Siano $(G, \star)$, $(G', \cdot)$ due gruppi. Sia $\phi : G \to G'$ un omomorfismo, quindi $\forall g, g' \in G, \phi(g \star g') = \phi(g) \cdot\phi(g')$.

Sappiamo che $Im(\phi) \equiv \phi(G) \leq G'$ _(visto in aula il 3/11/2023)_, quindi $\forall g_1, g_2 \in \phi(G), g_1 \cdot g^{-1}_2 \in \phi(G) \iff$
1. $\forall g_1, g_2 \in \phi(G), g_1 \cdot g_2 \in \phi(G)$
2. $\forall g \in \phi(G), g^{-1} \in \phi(G)$

#### 1.1 G commutativo
Sapendo che $G$ è commutativo, verificare che anche $\phi(G)$ è commutativo. Quindi va dimostrato che

$$\forall y_1, y_2 \in \phi(G), y_1 \cdot y_2 = y_2 \cdot y_1$$

Presi $y_1 = \phi(g_1), g_1 \in G$ e $y_2 = \phi(g_2), g_2 \in G$, si ha che 
- $\phi(g_1) \cdot \phi(g_2) \in \phi(G)$ Per (1)
- $\phi(g_1) \cdot \phi(g_2) = y_1 \cdot y_2$ Per come sono stati presi $y_1$ e $y_2$

Usando la definizione di omomorfismo $\phi(g_1 \star g_2) = y_1 \cdot y_2 \implies$ (usando la commutatività di $G$) $\implies \phi(g_2 \star g_1) = y_1 \cdot y_2 \implies$ (usando la definizione di omomorfismo) $\implies \phi(g_2) \cdot \phi(g_1) = y_1 \cdot y_2 \implies y_2 \cdot y_1 = y_1 \cdot y_2$

#### 1.2 Trasformato di un sottogruppo di G

Dato $(H, \star) \mid H \leq G$, verificare che $\phi(H) \leq G'$. Sapendo che $\phi(H) \subseteq \phi(G) \subseteq G'$, bisogna dimostrare che 

$$
\forall h_1, h_2 \in H, h_1 \star h^{-1}_2 \in H \leq G \implies \forall g_1, g_2 \in \phi(H), g_1 \diamond g^{-1} _2 \in \phi(H)
$$

 È noto che, dati due gruppi $(G, \star)$, $(H, \cdot)$ e preso $\phi : G \to H \mid \forall g_1, g_2 \in G, \phi(g_1 \star g_2) = \phi(g_1) \cdot \phi(g_2) \implies \phi(G) \leq H$ _(visto in aula il 3/11/2023)._ Sapendo che $H \leq G$, quindi $H$ ha le caratteristiche di un gruppo, allora $\phi(H)$ è un sottogruppo di $\phi(G)$ che è a sua volta sottogruppo di $G'$.
 
 > TODO: da rivedere, non pare abbastanza soddisfacente
 
 #### 1.3 Nucleo di $\phi$
 
 Verificare che $\ker\phi = \phi^{-1}(1_{G'}) = \\{g \in G \mid \phi(g) = 1_{G'}\\} \leq G$, quindi $\forall g_1, g_2 \in \ker\phi, g_1 \star g^{-1}_2 \in \ker\phi$

Si prendano $g_1, g_2 \in G \mid \phi(g_1) = 1\_{G'}, \phi(g_2) = 1\_{G'}$, bisogna dimostrare che $\phi(g_1 \star g^{-1}_2) = 1\_{G'}$, ovvero $g_1 \star g^{-1}_2 \in\ker\phi$. Si ha $\phi(g_1) \cdot \phi(g_2) = 1\_{G'}$ dato che $1\_{G'} \cdot 1\_{G'} = 1\_{G'}$, e $\phi(g_2)^{-1} = 1\_{G'}^{-1} = 1\_{G'} = \phi(g_2)$ quindi $\phi(g_1) \cdot (\phi(g_2))^{-1} = 1\_{G'}$, ma $\phi(g_2)^{-1} = \phi(g_2^{-1})$, per cui $\phi(g_1) \cdot \phi(g_2^{-1}) = \phi(g_1 \star g_2^{-1}) = 1\_{G'} \implies g_1 \star g_2^{-1} \in \ker\phi \\ \square$

**Comment by 18xChris** [comment]
> Siano (G,⋆), (G′,⋅) due gruppi. Sia ϕ:G→G′ un omomorfismo, quindi ∀g,g′∈G,ϕ(g⋆g′)=ϕ(g)⋅ϕ(g′).
> 
> Sappiamo che Im(ϕ)≡ϕ(G)≤G′ _(visto in aula il 3/11/2023)_, quindi ∀g1,g2∈ϕ(G),g1⋅g2−1∈ϕ(G)⟺
> 
>     1. ∀g1,g2∈ϕ(G),g1⋅g2∈ϕ(G)
> 
>     2. ∀g∈ϕ(G),g−1∈ϕ(G)
> 
> 
> #### 1.1 G commutativo
> 
> Sapendo che G è commutativo, verificare che anche ϕ(G) è commutativo. Quindi va dimostrato che
> 
> ∀y1,y2∈ϕ(G),y1⋅y2=y2⋅y1
> 
> Presi y1=ϕ(g1),g1∈G e y2=ϕ(g2),g2∈G, si ha che
> 
>     * ϕ(g1)⋅ϕ(g2)∈ϕ(G) Per (1)
> 
>     * ϕ(g1)⋅ϕ(g2)=y1⋅y2 Per come sono stati presi y1 e y2
> 
> 
> Usando la definizione di omomorfismo ϕ(g1⋆g2)=y1⋅y2⟹ (usando la commutatività di G) ⟹ϕ(g2⋆g1)=y1⋅y2⟹ (usando la definizione di omomorfismo) ⟹ϕ(g2)⋅ϕ(g1)=y1⋅y2⟹y2⋅y1=y1⋅y2
> #### 1.2 Trasformato di un sottogruppo di G
> 
> Dato (H,⋆)∣H≤G, verificare che ϕ(H)≤G′. Sapendo che ϕ(H)⊆ϕ(G)⊆G′, bisogna dimostrare che
> 
> ∀h1,h2∈H,h1⋆h2−1∈H≤G⟹∀g1,g2∈ϕ(H),g1⋄g2−1∈ϕ(H)
> 
> È noto che, dati due gruppi (G,⋆), (H,⋅) e preso ϕ:G→H∣∀g1,g2∈G,ϕ(g1⋆g2)=ϕ(g1)⋅ϕ(g2)⟹ϕ(G)≤H _(visto in aula il 3/11/2023)._ Sapendo che H≤G, quindi H ha le caratteristiche di un gruppo, allora ϕ(H) è un sottogruppo di ϕ(G) che è a sua volta sottogruppo di G′.
> 
> > TODO: da rivedere, non pare abbastanza soddisfacente
> 
> #### 1.3 Nucleo di
> 
> Verificare che ker⁡ϕ=ϕ−1(1G′)={g∈G∣ϕ(g)=1G′}≤G, quindi ∀g1,g2∈ker⁡ϕ,g1⋆g2−1∈ker⁡ϕ
> 
> Si prendano g1∈G∣ϕ(g1)=1G′ e g2∈G∣ϕ(g2)=1G′, bisogna dimostrare che ϕ(g1⋆g2−1)=1G′, ovvero g1⋆g2−1∈ker⁡ϕ. Quindi si ha ϕ(g1)⋅ϕ(g2)=1G′⟹ϕ(g1)⋅(ϕ(g2))−1=ϕ(g1)⋅ϕ(g2−1)=ϕ(g1⋆g2−1)=1G′⟹g1⋆g2−1∈ker⁡ϕ ◻
-----------------------------------------------------

Per  quanto riguarda 1.3 non è penso ia molto chiaro  perché ϕ(g1)⋅ϕ(g2)=1G′⟹ϕ(g1)⋅(ϕ(g2))−1=ϕ(g1)⋅ϕ(g2−1)=ϕ(g1⋆g2−1)=1G′

Io l'ho dimostrata così

 
![Screenshot from 2023-11-07 11-54-29](https://github.com/sapienzastudentsnetwork/algebra2324/assets/54262504/c53089db-7499-4ca1-85d1-8fa686903be5)

**Comment by CiottoloMaggico** [comment]
Punto 1 e 3 analogo a @CuriousCI 
Per il punto 2 ho preferito dimostrare che ϕ(H) < G' utilizzando la definizione "completa" di sottogruppo
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/5e4cefe8-e8ed-496c-b1e0-43dfd170f8c4)

**Comment by Elia-Belli** [comment]
- **(1)** Siano $g_1,g_2 \in G\Rightarrow g_1\star g_2=g_2\star g_1$ per commutatività di $G$
Ma allora $\phi(g_1\star g_2)=\phi(g_2\star g_1)\Rightarrow \phi(g_1)\star \phi(g_2)=\phi(g_2)\star\phi(g_1)\ \square$

- **(2)** Sia $H\leq G$ dimostrare che $\phi(H)\leq G'$:
Per ipotesi $H\subseteq G$ allora siccome $\phi(g_1\star g_2)=\phi(g_1)\star \phi(g_2),\forall g_1,g_2\in G$ ciò vale anche $\forall h_1,h_2\in H\subseteq G$
Quindi abbiamo $\phi:H\rightarrow G'$ omomorfismo di gruppi ($H$ (sotto)gruppo per ipotesi)
$\Rightarrow \phi(H)\leq G'\ \square$ (abbiamo dimostrato in precedenza che l'immagine di un omomorfismo è un sottogruppo) 

- ### Soluzione del prof al 1.3
   $Ker \phi=\\{g\in G|\phi(g)=1_{G'}\\}=\phi^{-1}(1_{G'})$ è un sottogruppo, infatti:
Scegliamo $g_1,g_2\in ker \phi$, ovvero $\phi(g_1)=1_{G'}=\phi(g_2)$
$\phi(g_1\star g_2^{-1})=\phi(g_1)\star\phi(g_2^{-1})=\phi(g_1)\star(\phi(g_2))^{-1}=1_{G'}\star (1_{G'})^{-1}=1_{G'}\in Ker\phi$
Allora per il _**criterio dei sottogruppi**_ $Ker\phi$ è un sottogruppo di $G\ \square$

**Comment by daromacarlo** [comment]
[esercizio 1 foglio 5.pdf](https://github.com/sapienzastudentsnetwork/algebra2324/files/13628750/esercizio.1.foglio.5.pdf)
